sdk/tables/data-tables/src/utils/continuationToken.ts

```diff
@@ -5,26 +5,26 @@ import { base64Decode, base64Encode } from "./bufferSerializer";
 
 export interface ContinuationToken {
   nextPartitionKey: string;
-  nextRowKey: string;
+  nextRowKey?: string;
```

Comment:
would not this be a breaking change? if a client code was accessing this property, it will not compile with this change because it will have to check if is present first.

Reply:
or is it not part of the public APIs?

Reply:
> or is it not part of the public APIs?

I don't believe it is part of the public API.  I didn't see any reference to it in `review\data-tables.api.md` and also we don't see a diff to that file as part of this PR (which I would expect if we were changing the public surface area)

Reply:
It is not part of the public API it is just being used internally. But good point, since it is not used outside of this module I'll remove the export